torchrl/modules/models/multiagent.py

```diff
@@ -30,6 +32,8 @@ def __init__(
         centralised: bool,
         share_params: bool,
         agent_dim: int,
+        num_inputs: int = 1,
+        num_outputs: int = 1,
```

Comment:
With these two things you mean cardinality of the inputs and outputs?

This is confusing with `n_agent_inputs` and outputs. I suggest to call it smth that contains "cardinality" like `input_cardinality`

Reply:
Do the names `input_tuple_len` and `output_tuple_len` make sense to you? The purpose of these arguments to allow a multi-agent net to take in and output a tuple. In the LSTM case it's a 2-len tuple `(input, (h_x, c_x))` and for GRU it's a 2-len tuple `(input, h_x)`.